commands.py

```diff
@@ -0,0 +1,102 @@
+from SchemaKeys import SchemaKeys
```

Comment:
this comment for the previous files
what are those files? and why did you push those files to the main?
I think you should not.

Reply:
what main? If you mean the main branch I'm quite sure I deleted all files in it.

Reply:
Those files are bytecode-compiled version of the program's files. Python creates them whenever I run the code. I guess they make running a little faster, as the lines that were compiled before don't need to be recompiled every time I run the program so they are kept in this cashe files, you can read more about it.
You just ignore them, and you are right I shouldn't push them again.